distributed/deploy/ssh.py

```diff
@@ -139,6 +142,9 @@ async def start(self):
         logger.debug("%s", line)
         await super().start()
 
+    def __repr__(self):
+        return f"<SSH {type(self).__name__}: status={self.status}>"
+
```

Comment:
This can also be removed as they are identical to the `__repr__` on `Process`.

Reply:
I also changed the typename to the  `dask.utils.typename(type(self))`
![image](https://user-images.githubusercontent.com/62539811/129599524-877df2fb-55a5-48b9-bf7f-4f847d017d22.png)

Is this a good change?
It was "SSH Worker" and "SSH Scheduler" before.

Reply:
I have no strong preference here.

I would suggest updating the status to be a string so that it looks like `status="running"`. I'm not sure showing the `Status` object here has much value.

Reply:
Oh yes, that is shorted and cleaner.